experimental/kubernetes/spinnaker-local.yml

```diff
@@ -7,12 +7,7 @@ providers:
     # If you want to deploy some services to Amazon Web Services (AWS),
     # set enabled and provide primary credentials for deploying.
     # Enabling AWS is independent of other providers.
-    enabled: true
-    defaultRegion: us-east-1
-    defaultIAMRole: BaseIAMRole
-    primaryCredentials:
-      name: my-aws-account
-      # Store actual credentials in /home/spinnaker/.aws/credentials
+    enabled: false
```

Comment:
I'd say keep it simple - there's already a lot going on in this file and additional, disabled junk just makes it harder to understand what's really going on.